docker-compose.yml

```diff
@@ -4,7 +4,7 @@ version: '3.8'
 # On re-starts it will contain data(and not execute scripts) but on re-builds it will not.
 services:
   postgres:
-    image: postgres:9.6
+    image: postgres:14.5-alpine
```

Comment:
This has to be 14 specifically because that is the exact version used.

Reply:
done.